tests/utils/k8s/k8s.go

```diff
@@ -128,3 +128,25 @@ func ExecInPodContainer(
 
 	return stdout, nil
 }
+
+func FetchLogsOfPod(namespaceName, podName, containerName string) (*bytes.Buffer, error) {
```

Comment:
Would `PodContainerLogs` be a more representative name for this function?

Reply:
Let's go with `FetchContainerLogs`

Reply:
How is that function different than the [already existing `k8s.GetPodContainerLogs`](https://github.com/mesosphere/kudo-cassandra-operator/blob/8932a7e5f58b52495466ca16265aace4b8b3ca97/tests/utils/k8s/k8s.go#L75) by the way?

Reply:
It's not,  I made a duplicate by mistake :thinking:! Thanks for pointing it out, will fix that